src/README.md

```diff
@@ -14,6 +14,12 @@ It’s important to note the differences between the framework and the CMS. Some
 [![Documentation](https://readthedocs.org/projects/orchardcore/badge/)](https://docs.orchardcore.net/)
 [![Crowdin](https://badges.crowdin.net/orchard-core/localized.svg)](https://crowdin.com/project/orchard-core)
 
+#### Local communities
+
+中文资源
+
+[![Orchard Core CN 中文讨论组](https://pub.idqqimg.com/wpa/images/group.png)](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)
+
```

Comment:
I just did merge this. Please remove. Also irrelevant for this PR based on the title of the PR.

Reply:
Removed

It's strange that the latest code I got from the main branch didn't get Update of https://github.com/OrchardCMS/OrchardCore/pull/8314/files